components/admin_console/team_channel_settings/channel/details/channel_profile.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Channel} from 'mattermost-redux/types/channels';
+import {Team} from 'mattermost-redux/types/teams';
+
+import {t} from 'utils/i18n';
+
+import AdminPanel from 'components/widgets/admin_console/admin_panel';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
+
+interface ChannelProfileProps {
+    channel: Partial<Channel>;
+    team: Partial<Team>;
+}
+
+export const ChannelProfile: React.SFC<ChannelProfileProps> = (props: ChannelProfileProps): JSX.Element => {
+    const {team, channel} = props;
+    return (
+        <AdminPanel
+            id='channel_profile'
+            titleId={t('admin.channel_settings.channel_detail.profileTitle')}
+            titleDefault='Channel Profile'
+            subtitleId={t('admin.channel_settings.channel_detail.profileDescription')}
+            subtitleDefault='Summary of the channel, including the channel name.'
+            className='channel_profile'
```

Comment:
Same here with the className

Reply:
yes so its a required prop in admin_panel.tsx   should i just change the props to className to be optional?

Reply:
@allenlai18 Yes, it makes sense to change it to optional